okonf/facts/systemd.py

```diff
@@ -39,3 +39,20 @@ async def enforce(self, host: Executor) -> bool:
         async with host.lock(f"systemctl-start-service-{self.name}"):
             await host.run(f"sudo systemctl start {self.name}")
         return True
+
+
+class ServiceRestarted(Fact):
+    """Restart a Systemd service."""
```

Comment:
Use "Ensure that a Systemd service is restarted" to follow the same style as the other facts.